src/stratum_server.cpp

```diff
@@ -847,6 +847,31 @@ void StratumServer::on_share_found(uv_work_t* req)
 		const char* s = client->m_customUser;
 		LOGINFO(0, log::Green() << "SHARE FOUND: mainchain height " << height << ", diff " << sidechain_difficulty << ", client " << static_cast<char*>(client->m_addrString) << (*s ? " user " : "") << s << ", effort " << effort << '%');
 		pool->submit_sidechain_block(share->m_templateId, share->m_nonce, share->m_extraNonce);
+
+		if(!pool->params().m_onShareFound.empty())
+		{
+		    std::string cmd = pool->params().m_onShareFound;
+		    //Add what was found
+		    cmd += " SHARE";
+		    //Add the client address
+		    cmd = cmd + " " + client->m_addrString;
+		    //Add the User
+		    cmd = cmd + " " + client->m_customUser;
+		    //The block height
+		    cmd = cmd + " " + std::to_string(height);
+		    //The Difficulty
+		    cmd = cmd + " " + std::to_string(sidechain_difficulty.to_double());
+		    
+		    //system() requires char and not string, convert to char
+		    char sys[cmd.length() + 1];
+		    strcpy(sys, cmd.c_str());
+
+		    //If there is an error, then log it
+		    if(!system(sys))
```

Comment:
Same, create cross-platform function in util.cpp

Reply:
Currently working on.

Reply:
https://github.com/monero-project/monero/blob/master/src/common/spawn.cpp#L52 is a good reference